checks/http/sap.rb

```diff
@@ -63,23 +63,23 @@ def generate_checks(url)
               type: 'fingerprint',
               require_product: 'NetWeaver',
               category: 'application',
-              tags: ['ApplicationServer'],
+              tags: ['Application Server'],
               vendor: 'SAP',
               product: 'NetWeaver',
               description: 'netweaver version in irj/portal',
               version: nil,
               references: [''],
               match_type: :content_title,
-              match_content: /SAP NetWeaver ApplicationServer/i,
-              # SAP NetWeaver ApplicationServer 7.53
-              dynamic_version: ->(x) { _first_header_capture(x, /SAP NetWeaver ApplicationServer ([\d.]+)/i) },
+              match_content: /SAP NetWeaver Application Server/i,
```

Comment:
same here, fingerprint broken?

Reply:
This was broken in a previous commit I did, so I adjusted them back.